coreos/lostromos
# Fix broken test import

# What Are We Doing Here

This fixes a broken import in the tests and re-runs `dep ensure` to make sure that our dependencies can be resolved.

## How to Verify

1. Check out this PR
2. Run `dep ensure`
3. Profit
